os/board/rtl8720e/src/component/soc/amebalite/fwlib/ram_km4/ameba_app_start.c

```diff
@@ -43,15 +43,15 @@ extern u32 GlobalDebugEnable;
 #ifdef CONFIG_PLATFORM_TIZENRT_OS
 extern unsigned int __StackStart;
 extern unsigned int __StackLimit;
-extern unsigned int _ebss;
+extern uint8_t __psram_heap_buffer_start__[];
```

Comment:
Kindly check for build break in tc config >
/root/tizenrt/os/arch/arm/src/board/libboard.a(ameba_app_start.o): In function `os_heap_init':
/root/tizenrt/os/board/rtl8720e/src/component/soc/amebalite/fwlib/ram_km4/ameba_app_start.c:71: undefined reference to `__psram_heap_buffer_start__'
make[1]: *** [../build/output/bin/tinyara.axf] Error 1

Reply:
I did not encounter any build errors with TC config.
May I know how did you get this error?

Reply:
The code was not rebased. It seems to work fine. We can ignore this comment